problems/strings.js

```diff
@@ -103,15 +155,32 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+function disemvowelLongWords(string) {
+  newString = "";
+  let vowels = "aeiouAEIOU"
+  let words = string.split(" "); 
+  for(let i = 0; i <= words.length; i++) {
```

Comment:
This question is coming up wrong because your loop goes until the length, but the length is always 1 longer than the index